homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -117,7 +121,10 @@ async def async_update(self) -> None:
 
     async def async_turn_on(self) -> None:
         """Turn on the media player."""
-        await self._remote.async_turn_on(context=self._context)
+        if self._turn_on:
+            await self._turn_on.async_run(self.hass, self._context)
+        else:
+            await self._remote.async_turn_on()
```

Comment:
This means the remote service will fire triggers instead of calling the API to turn on?